Detectors/TPC/workflow/src/FileWriterSpec.cxx

```diff
@@ -47,13 +47,40 @@ using o2::dataformats::FileMetaData;
 using SubSpecificationType = DataAllocator::SubSpecificationType;
 using DetID = o2::detectors::DetID;
 
+enum class BranchType {
+  Krypton,
+  Digits,
+};
+
+const std::unordered_map<std::string, BranchType> BranchTypeMap{
+  {"krypton", BranchType::Krypton},
+  {"digits", BranchType::Digits},
+};
+
+const std::unordered_map<BranchType, std::string> BranchName{
+  {BranchType::Krypton, "TPCBoxCluster"},
+  {BranchType::Digits, "TPCDigit"},
+};
+
+const std::unordered_map<BranchType, std::string> TreeName{
+  {BranchType::Krypton, "Clusters"},
+  {BranchType::Digits, "o2sim"},
```

Comment:
@wiechula there are extra `,` at the end of each block. The compiler does not complain but I think it is better to remove them.

Reply:
@shahor02 as far as I know it is allowed since C99. I would like to keep it, sine it looks more symmetric and makes adding new elements a bit easier.

Reply:
ok, thought looks a bit awkward...